src/validationFunctions.js

```diff
@@ -1,58 +1,72 @@
 
-    // SIMPLE JS FUNCTIONS: 
-    function isInvalidString (value) {
-        return typeof value !== "string"                                  
-    }
+// SIMPLE JS FUNCTIONS: 
+function isInvalidString(value) {
```

Comment:
`isInvalidString/isInvalidNum` are really good candidates for unit tests, so it'd be good to add a couple now that they're extracted into their own file.